src/components/faction-toggle.module.css

```diff
@@ -0,0 +1,27 @@
+.faction-filter {
+  padding: 0.5rem;
+}
+
+.faction-filter .color-active-guardian[data-state="on"] {
```

Comment:
I'm sorry if I missed this (the mobile UI sucks a bit), but we can also remove these declarations from `filters.module.css` now.